### DISCUSSION TITLE
3rd Party Import Web Interface

### DISCUSSION THREAD

**Discussion by mattheys** [beetbox/beets#4383] [General]
Does anyone know of a 3rd party web interface I can use to run an import?  It would be great if I could do it on my mobile while sat with the wife on the sofa instead of locked away in my computer room on my own going through all the matches.

If nothing already exists can I use before_choose_candidate or import_task_before_choice to hook in and get the choices, relay it to a web site and then return the choice?

**Comment by sampsyo** [comment]
Unfortunately, I don't think anything like that exists (although I would be happy to be proven wrong!). We have discussed ways to do this in #3073 and #718 previously, and probably elsewhere. But providing enough hooks to enable a good, asynchronous UI for the importer would be tricky!
